src/dodal/devices/focusing_mirror.py

```diff
@@ -0,0 +1,145 @@
+import time
+from enum import Enum
+
+from ophyd import Component, Device, EpicsMotor, EpicsSignal
+from ophyd.status import Status
+
+from dodal.log import LOGGER
+
+VOLTAGE_POLLING_DELAY_S = 0.5
+
+# The default timeout is 60 seconds as voltage slew rate is typically ~2V/s
+DEFAULT_SETTLE_TIME_S = 60
+
+DEMAND_ACCEPTED_OK = 1
+
+
+class MirrorStripe(Enum):
+    RHODIUM = "Rhodium"
+    BARE = "Bare"
+    PLATINUM = "Platinum"
+
+
+class MirrorVoltageDevice(Device):
+    _actual_v: EpicsSignal = Component(EpicsSignal, "R")
+    _setpoint_v: EpicsSignal = Component(EpicsSignal, "D")
+    _demand_accepted: EpicsSignal = Component(EpicsSignal, "DSEV")
+
+    def set(self, value, *, timeout=None, settle_time=0, **kwargs):
```

Comment:
Should: My understanding on what this function is doing is:
* Set the setpoint
* Periodically poll the `demand_accepted` signal until it is 1

One issue here may be that is it possible the demand accepted is still a stale 1 from the previous value?

In any case, I think a cleaner solution is to use [subscribe](https://nsls-ii.github.io/ophyd/_as_gen/ophyd.ophydobj.OphydObject.subscribe.html) on the `demand_accepted` and wait for the event where the value changes from 0 -> 1. One potential gotcha on this is you want to set the subscription up before you do the set, to avoid race conditions. There's some precedent on doing this [here](https://github.com/DiamondLightSource/dodal/blob/f5fba125239e8dab128f13cbb7fe5aa080570c01/src/dodal/devices/fast_grid_scan.py#L179)